staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -1223,7 +1223,7 @@ type SecretVolumeSource struct {
 }
 
 const (
-	SecretVolumeSourceDefaultMode int32 = 0644
+	SecretVolumeSourceDefaultMode int32 = 0o644
```

Comment:
hmm changes to `*VolumeSourceDefaultMode` were done by accident, but they seem correct, should I keep them?